system/mp/mpcomm.cpp

```diff
@@ -1120,8 +1153,10 @@ protected: friend class CMPPacketReader;
                 try {
                     s->shutdown();
                 }
-                catch (IException *) { 
+                catch (IException *e) {
                     socketfailed = true; // ignore if the socket has been closed
+                    WARNLOG("closeSocket() : Ignoring shutdown error");
```

Comment:
For now.
We I don't think we'll want to keep all the changes in this PR once/if we get to the bottom of this issue.